sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -736,8 +757,31 @@ bool DDSE_dict_init(THD *thd, dict_init_mode_t dict_init_mode, uint version) {
   List<const Plugin_tablespace> ddse_tablespaces;
   if (ddse->ddse_dict_init == nullptr ||
       ddse->ddse_dict_init(dict_init_mode, version, &ddse_tables,
-                           &ddse_tablespaces))
+                           &ddse_tablespaces)) {
+    LogErr(ERROR_LEVEL, ER_LOG_PRINTF_MSG,
+           "Calling DDSE handlerton ddse_dict_init failed");
     return true;
+  }
+
+  /*
+    Always call innodb handler API unless disabled
+    innobase_dict_init will initialize these innodb specific tables
+  */
+  if (ddse->db_type != DB_TYPE_INNODB) {
+    handlerton *innodb_ddse = ha_resolve_by_legacy_type(thd, DB_TYPE_INNODB);
```

Comment:
InnoDB is not the DDSE, thus might be clearer to rename the local var:

```suggestion
    handlerton *innodb_se = ha_resolve_by_legacy_type(thd, DB_TYPE_INNODB);
```